content/integrations/kui.md

```diff
@@ -25,11 +25,15 @@ To run the KUI Terminal, use:
 ```sh
 npm start
 ```
+To install iter8, refer to the [iter8 installation guidelines](https://preliminary.iter8.tools/installation/).
 
 ## Currently available commands
 
-You can use the following commands once the KUI terminal is up and running:
-1. `iter8 metrics`: This command opens a KUI sidecar where the you can perform CRUD operations on the iter8 metric configmap. Specifically, you can add, edit, delete and restore metrics on the KUI sidecar that is opened. A sample image of the output is as follows:
+You can use the following commands once the KUI terminal is up and iter8 has been installed:
+
+#### iter8 metrics
```

Comment:
Perhaps still make code:
Use back quotes around iter8 metrics (and other headings)